### PR TITLE
Add enrich span callable for sqlalchemy trace

### PR DESCRIPTION
**Pull request by avi-city**
Hi,

Following issue #56 , since we needed to have such an option, I forked the library and implemented the `enrich_span` you suggested.
Let me know what you think.

Thanks,
Avi

### CODE REVIEW

File: opentracing_utils/libs/_sqlalchemy.py [hunk 1]
```diff
@@ -28,6 +28,9 @@ def trace_sqlalchemy(operation_name=None, span_extractor=None, set_error_tag=Fal
     :param skip_span: Callable to determine whether to skip this SQL query(ies) spans. If returned ``True`` then span
                       will be skipped.
     :type skip_span: Callable[conn, cursor, statement, parameters, context, executemany]
+
+    :param enrich_span: Callable to enrice the span with additional data.
```
Comment (mohabusama) [inline opentracing_utils/libs/_sqlalchemy.py:32]:
```suggestion
    :param enrich_span: Callable to enrich the span with additional data.
```

Comment (avi-city) [reply opentracing_utils/libs/_sqlalchemy.py]: Oops 😅

**Comment by mohabusama** [comment]
@avi-city LGTM. I would suggest updating the README SqlAlchemy examples with a usage example of the new option. Can be right under the skip_span example.

**Comment by avi-city** [comment]
Added

**Comment by mohabusama** [comment]
👍

**Comment by mohabusama** [comment]
@pitr @vetinari can we merge this one and publish a new release?

**Comment by pitr** [comment]
:+1:

**Pull request merged**